server/claim-review/dto/create-claim-review.dto.ts

```diff
@@ -9,15 +9,15 @@ import { Personality } from "../../personality/schemas/personality.schema";
 import { Claim } from "../../claim/schemas/claim.schema";
 
 export enum ClassificationEnum {
-    "not-fact",
-    "true",
-    "true-but",
-    "arguable",
-    "misleading",
+    "not-fact" = 1,
     "false",
+    "misleading",
     "unsustainable",
-    "exaggerated",
     "unverifiable",
+    "exaggerated",
+    "arguable",
+    "true-but",
```

Comment:
Let's take this opportunity to rename true to `trustworthy`. We already that for i18n labels, but we should change for enums too.